src/container/rpc.h

```diff
@@ -113,6 +116,8 @@ enum cont_operation {
 
 extern struct crt_proto_format cont_proto_fmt;
 
+#define CONT_OP_DUMMY_LABEL "NO LABEL, OPENING BY UUID"
```

Comment:
This is not used. I think I had put it here early during the editing of the patch thinking it may be needed in the implementation.